ccd-definition/CaseEvent/AnyState.json

```diff
@@ -0,0 +1,13 @@
+{
+  "LiveFrom": "01/01/2017",
+  "CaseTypeID": "CARE_SUPERVISION_EPO",
+  "ID": "handleSupplementaryEvidence",
+  "Name": "Handle supplementary evidence",
+  "Description": "Handle supplementary evidence",
+  "PreConditionState(s)": "*",
```

Comment:
Note that this means that it will be visible in the deleted state

Reply:
only solicitor has access to deleted state